modules/tinymce/src/plugins/table/test/ts/browser/ModifyColumnsTableResizeTest.ts

```diff
@@ -133,6 +133,28 @@ describe('browser.tinymce.plugins.table.ModifyColumnsTableResizeTest', () => {
           mceTableDeleteCol: 0
         });
       });
+
+      it('TINY-6711: will function with a colspan', () => {
```

Comment:
We can remove the comment on line 55 now

Reply:
There's a number of additional more complex colspan tests we should include, so that's why I left that (eg colspan with colgroups).